apps/system/style/action_menu/action_menu_extended.css

```diff
@@ -136,3 +136,9 @@ form[role="dialog"][data-type="object"].visible {
     z-index: -1;
   }
 }
+
+/* RTL View */
+html[dir="rtl"] form[role="dialog"][data-type="action"] > menu > button.icon,
+html[dir="rtl"] form[role="dialog"][data-type="object"] > menu > button.icon {
+  background-position: right 0.5rem center;
+}
```

Comment:
nit: can we move this definition to under the non-rtl `button.icon` definition above [1]?

1.) https://github.com/mozilla-b2g/gaia/pull/27007/files#diff-276bfb67dfd3983bd8de40144517b791R33